Readability.js

```diff
@@ -329,10 +329,25 @@ Readability.prototype = {
   },
 
   _setNodeTag: function (node, tag) {
-    // FIXME this doesn't work on anything but JSDOMParser (ie the node's tag
-    // won't actually be set).
-    node.localName = tag.toLowerCase();
-    node.tagName = tag.toUpperCase();
+    this.log("_setNodeTag", node, tag);
+    if (node.__JSDOMParser__) {
```

Comment:
JSDOMParser doesn't seem to support `ownerDocument`, though that's probably no bof deal adding support for it.